app.rb

```diff
@@ -22,12 +22,17 @@ def initialize
     @compliance_data = ComplianceData.new
   end
 
+  cloudgov_path = File.join(root, 'node_modules', 'cloudgov-style')
   set :assets, Sprockets::Environment.new(root)
 
   configure do
     assets.append_path File.join(root, 'assets', 'stylesheets')
     assets.append_path File.join(root, 'assets', 'javascripts')
 
+    # cloudgov-style css and js. must be installed before running via `npm install`
+    assets.append_path File.join(cloudgov_path, 'css')
+    assets.append_path File.join(cloudgov_path, 'js')
```

Comment:
Not sure we need to add the `cg-style` js files, as they only include analytics at this point. Would be good to know if we want to track the same users for compliance viewer with this same GA #.